Library/ViewModels/PledgeViewModel.swift

```diff
@@ -128,6 +129,31 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       refTag
     ).map { $0 as CreateBackingData }
 
+    // MARK: Create Backing
+
+    let createBackingEvent = Signal.combineLatest(backingData, self.creditCardSelectedSignal)
+      .takeWhen(self.pledgeButtonTappedSignal)
+      .map { backingData, paymentSourceId in
+        (
+          backingData.project,
+          backingData.reward,
+          backingData.pledgeAmount,
+          backingData.selectedShippingRule,
+          backingData.refTag,
+          paymentSourceId
+        )
+      }
+      .map(CreateBackingInput.input(from:reward:pledgeAmount:selectedShippingRule:refTag:paymentSourceId:))
+      .switchMap { input in
+        AppEnvironment.current.apiService.createBacking(input: input)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
```

Comment:
Notice that we don't care about the result here - since we are moving to a synchronous `createBacking` mutation, any errors in creating the pledge *should* come back as regular graph errors.